From 829968ed2ace38908db3ead8204d544d7198159f Mon Sep 17 00:00:00 2001
From: Danny Kukawka <danny.kukawka@web.de>
Date: Thu, 11 Mar 2010 13:48:03 +0100
Subject: [PATCH 35/45] updated SPEC due to new properties from commit dea5997df896

Updated SPEC due to new properties from commit dea5997df896.
Added new:
- scsi.hotpluggable
- scsi_host.hotpluggable
---
 doc/spec/hal-spec-properties.xml |   16 ++++++++++++++++
 1 files changed, 16 insertions(+), 0 deletions(-)

diff --git a/doc/spec/hal-spec-properties.xml b/doc/spec/hal-spec-properties.xml
index fdfacf1..190e93f 100644
--- a/doc/spec/hal-spec-properties.xml
+++ b/doc/spec/hal-spec-properties.xml
@@ -2755,6 +2755,14 @@ org.freedesktop.Hal.Device.Volume.method_signatures = {'ssas', 'as', 'as'}
             </row>
             <row>
               <entry>
+                <literal>scsi.hotpluggable</literal> (bool)
+              </entry>
+              <entry></entry>
+              <entry>No</entry>
+              <entry>The SCSI device can be removed while the system is running (e.g. in case of eSATA)</entry>
+            </row>
+            <row>
+              <entry>
                 <literal>scsi.type</literal> (string)
               </entry>
               <entry>Example: disk</entry>
@@ -2856,6 +2864,14 @@ org.freedesktop.Hal.Device.Volume.method_signatures = {'ssas', 'as', 'as'}
               <entry>Yes</entry>
               <entry>A unique number identifying the SCSI host adaptor</entry>
             </row>
+            <row>
+              <entry>
+                <literal>scsi_host.hotpluggable</literal> (bool)
+              </entry>
+              <entry></entry>
+              <entry>No</entry>
+              <entry>The host device can be removed while the system is running</entry>
+            </row>
           </tbody>
         </tgroup>
       </informaltable>
-- 
1.7.2.1

